src/main/java/uk/co/haxyshideout/chococraft2/config/Constants.java

```diff
@@ -1,18 +1,22 @@
 package uk.co.haxyshideout.chococraft2.config;
 
+import net.minecraft.entity.EntityAgeable;
+import net.minecraft.network.datasync.DataParameter;
+import net.minecraft.network.datasync.DataSerializers;
+import net.minecraft.network.datasync.EntityDataManager;
+
 /**
  * Created by clienthax on 12/4/2015.
  */
-public class Constants {
+public class Constants
+{
 	public static final String MODID = "chococraft2";
 	public static final String MODNAME = "ChocoCraft2";
 	public static final String MODVERSION = "0.0.2";
 
-	// for EntityTamable, 2-5, 13-15, 18-31 are usable
-	// http://www.minecraftforge.net/wiki/Datawatcher
-	public static final int dataWatcherVariant = 31;
-	public static final int dataWatcherBagType = 30;
-	public static final int dataWatcherSaddled = 29;
-	public static final int dataWatcherMale = 28;
-	public static final int dataWatcherFollowingOwner = 27;
+	public static final DataParameter<Byte> dataWatcherVariant = EntityDataManager.<Byte> createKey(EntityAgeable.class, DataSerializers.BYTE);
```

Comment:
@clienthax Some of those are shared between `EntityBabyChocobo` and `EntityChocobo`